sqlglot/dialects/presto.py

```diff
@@ -348,3 +348,13 @@ def transaction_sql(self, expression: exp.Transaction) -> str:
             modes = expression.args.get("modes")
             modes = f" {', '.join(modes)}" if modes else ""
             return f"START TRANSACTION{modes}"
+
+        def withingroup_sql(self, expression: exp.WithinGroup) -> str:
```

Comment:
should this instead be a transform? i think spark would use it as well

Reply:
looks like duckdb already does this transformation https://duckdb.org/docs/sql/aggregates.html

Reply:
expression.find(exp.Ordered)

Reply:
> should this instead be a transform? i think spark would use it as well

Sure

> looks like duckdb already does this transformation

Awesome, good to know.

> expression.find(exp.Ordered)

Wdym?

Reply:
ya

Reply:
Done